viz-lib/src/visualizations/chart/plotly/applyLayoutFixes.js

```diff
@@ -127,5 +117,7 @@ export default function applyLayoutFixes(plotlyElement, layout, options, updateP
         break;
       // no default
     }
+  } else {
+    updatePlot(plotlyElement, pick(layout, ["width", "height"]));
```

Comment:
This solves another issue that Charts without legends weren't updating when resized.